sass/includes/_people.scss

```diff
@@ -0,0 +1,51 @@
+@use "@nationalarchives/frontend/nationalarchives/tools/colour";
+
+.people {
+    margin-bottom: 3rem;
+
+    &__list {
+        margin: 0;
+        padding: 0;
+        display: flex;
+        flex-direction: column;
+        gap: 1.5rem;
+    }
+
+    &__person {
+        display: flex;
+        gap: 1.5rem;
+    }
+
+    &__image {
+        border-radius: 50%;
+        width: 5rem;
+        height: 5rem;
+        flex-grow: 0;
+        flex-shrink: 0;
+
+        // user icon fallback (font-awesome)
+        &--default {
+            background-color: $color__pastel-pink;
```

Comment:
Should this be flexible for different colours I wonder? I've been using the template modifier class (`--pink-accent`) in some of my styles though appreciate it might be overkill at this stage

Reply:
If you like, you can import the colour tools and use:

```scss
@include colour.light-accent-background;
```

That will then take the colour from the page's accent (https://github.com/nationalarchives/ds-wagtail/blob/39f0a1bc7de46bb0db4508a3a2093aae8852e135/templates/whatson/whats_on_page.html#L5)

Reply:
done.